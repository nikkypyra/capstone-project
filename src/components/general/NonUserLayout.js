import React from 'react'
import Header from './Header'

export default function NonUserLayout({ children }) {
  return (
    <>
      <Header />
      <main>{children}</main>
    </>
  )
}
