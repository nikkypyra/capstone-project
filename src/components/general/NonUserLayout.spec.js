import React from 'react'
import { render, cleanup } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import NonUserLayout from './NonUserLayout'

afterEach(cleanup)

describe('<NonUserLayout />', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <NonUserLayout />
      </MemoryRouter>
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
