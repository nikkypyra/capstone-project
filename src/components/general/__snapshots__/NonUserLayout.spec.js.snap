// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NonUserLayout /> should render correctly 1`] = `
<DocumentFragment>
  <header
    class="Header__HeaderStyled-ex8xpx-0 imCsBQ"
  >
    <img
      alt="pawlog"
      src="pawlog.png"
    />
  </header>
  <main />
</DocumentFragment>
`;
